presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -2845,4 +2846,17 @@ public FeaturesConfig setHandleComplexEquiJoins(boolean handleComplexEquiJoins)
         this.handleComplexEquiJoins = handleComplexEquiJoins;
         return this;
     }
+
+    public boolean isPartitionJoinIfProbeInSingleNode()
+    {
+        return partitionJoinIfProbeInSingleNode;
+    }
+
+    @Config("optimizer.partition-join-if-probe-in-single-node")
+    @ConfigDescription("Choose partitioned join if broadcast join has probe side input in single node")
+    public FeaturesConfig setPartitionJoinIfProbeInSingleNode(boolean partitionJoinIfProbeInSingleNode)
+    {
+        this.partitionJoinIfProbeInSingleNode = partitionJoinIfProbeInSingleNode;
+        return this;
+    }
```

Comment:
I think we should stop using featureconfig for very specific optimizations (that are off by default). Session params are easier and enough.

Reply:
how do we enable for specific clusters then?

Reply:
> how do we enable for specific clusters then?

Session props can also be set for specific clusters IIRC